app/services/inherited_proofing/va/service_payload_validator.rb

```diff
@@ -0,0 +1,74 @@
+require_relative 'field_validatable'
+
+module InheritedProofing
+  module Va
+    # Validates the payload_hash returned from the inherited proofing service.
+    class ServicePayloadValidator
+      include FieldValidatable
+
+      # TODO NOW: regex need to be verified and/or added
+      FIELD_TEMPLATE = {
+        first_name: { regex: /[a-zA-Z\s]/, required: true },
+        last_name: { regex: /[a-zA-Z\s]/, required: true },
+        phone: { regex: /[0-9]{9}/, required: false },
```

Comment:
If we are going to validate phone numbers this is a pretty naive validation. We have a pattern for validation of phones during identity verification: https://github.com/18F/identity-idp/blob/main/app/forms/idv/phone_form.rb#L9